### PR TITLE
10 packages from gitlab.com/nomadic-labs/resto/-/archive/v0.6/resto-v0.6.tar.gz

### PR DESCRIPTION
**Pull request by raphael-proust**
This pull-request concerns:
-`ezresto.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`ezresto-directory.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`resto.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`resto-acl.0.6`: Access Control Lists for Resto
-`resto-cohttp.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`resto-cohttp-client.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`resto-cohttp-self-serving-client.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`resto-cohttp-server.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs - server library
-`resto-directory.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs
-`resto-json.0.6`: A minimal OCaml library for type-safe HTTP/JSON RPCs

---
* Homepage: https://gitlab.com/nomadic-labs/resto
* Source repo: git+https://gitlab.com/nomadic-labs/resto
* Bug tracker: https://gitlab.com/nomadic-labs/resto/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/ezresto-directory/ezresto-directory.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+dev-repo: "git+https://gitlab.com/nomadic-labs/resto"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" { >= "4.08" }
+  "dune" { >= "1.7" }
```
Comment (kit-ty-kate) [inline packages/ezresto-directory/ezresto-directory.0.6/opam:16]: According to your `dune-project` file this needs to be >= 1.11. Could you modify it everywhere?

Comment (raphael-proust) [reply packages/ezresto-directory/ezresto-directory.0.6/opam:16]: Thanks for spotting that! I've made the changes and pushed anew.

**Comment by raphael-proust** [comment]
It looks like the failures are unrelated to the package itself. I'll rebase on master and force push to trigger the CI anew.

**Comment by raphael-proust** [comment]
The errors seem to not be related to the packages, but I'm not entirely sure. It's a whole lot of logs indicating that libev somehow didn't install correctly.

But I'm not sure, maybe the PR actually breaks some revdeps in subtle ways?

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**